translations/da/get5.phrases.txt

```diff
@@ -0,0 +1,383 @@
+    {
+        "da"            "til denne halvleg"
+    }
+    "MaxPausesUsedInfoMessage"
+    {
+        "#format"       "{1:d},{2:s}"
+        "da"            "Dit hold har allerede brugt max {1} pauser{2}."
+    }
+    "MaxPausesTimeUsedInfoMessage"
+    {
+        "#format"       "{1:d},{2:s}"
+        "da"            "Dit hold har allerede brugt max {1} sekunder af pause tid{2}."
+    }
+    "MatchPausedByTeamMessage"
+    {
+        "#format"       "{1:N}"
+        "da"            "{1} paused kampen."
+    }
+    "MatchTechPausedByTeamMessage"
+    {
+        "#format"       "{1:N}"
+        "da"            "{1} har bedt om en teknisk pause."
+    }
+    "AdminForceTechPauseInfoMessage"
+    {
+        "da"            "En admin har bedt om en teknisk pause."
+    }
+    "PauseTimeExpiration10SecInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "{1} har næsten ikke mere pause tid, unpauser om 10 sekunder."
+    }
+    "PauseTimeExpirationInfoMessage"
+    {
+        "#format"       "{1:s},{2:d},{3:s}"
+        "da"            "{1} har {2} sekunder af pause tid tilbage {3}."
+    }
+    "PauseRunoutInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "{1} er løbet tør for pause tid, unpauser kampen."
+    }
+    "MatchUnpauseInfoMessage"
+    {
+        "#format"       "{1:N}"
+        "da"            "{1:N} unpaused kampen."
+    }
+    "WaitingForUnpauseInfoMessage"
+    {
+        "#format"       "{1:s},{2:s}"
+        "da"            "{1} vil gerne unpause, venter på {2} til at skrive !unpause."
+    }
+    "PausesLeftInfoMessage"
+    {
+        "#format"       "{1:s},{2:d},{3:s}"
+        "da"            "{1} har {2} pauser tilbage {3}."
+    }
+    "OnePauseLeftInfoMessage"
+    {
+        "#format"       "{1:s},{2:d},{3:s}"
+        "da"            "{1} har {2} pauser tilbage{3}."
+    }
+    "TeamFailToReadyMinPlayerCheck"
+    {
+        "#format"       "{1:d}"
+        "da"            "Der skal være minimum {1} spiller(e) på serveren for at gøre klar."
+    }
+    "TeamReadyToVetoInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "{1} er klar til at veto."
+    }
+    "TeamReadyToRestoreBackupInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "{1} er klar til at genoprette match backup."
+    }
+    "TeamReadyToKnifeInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "{1} er klar til at knife om sider."
+    }
+    "TeamReadyToBeginInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "{1} er klar til at begynde kmapen."
+    }
+    "TeamNotReadyInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "da"            "{1} er ikke længere klar."
+    }
+    "ForceReadyInfoMessage"
+    {
+        "#format"       "{1:d}"
+        "da"            "Du kan skrive {GREEN}!forcready {NORMAL} for tvinge dit hold klar hvis der er mindre end {GREEN}{1}{NORMAL} spillere."
```

Comment:
you are missing e on forcready = forceready

Reply:
Thats my fault.. Was maybe a tad to quick.